client/assets/js/snake.js

```diff
@@ -152,4 +154,12 @@ export default class Snake {
 			}
 		});
 	}
+
+    removeSnakeFromScreen(x, y, width) {
+        this.context.beginPath();
+        this.context.rect(x, y, width, width);
```

Comment:
if both values are the same, it probably should be named `size` instead of `width`